src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -99,5 +99,9 @@ def load_command_table(self, _):
         g.custom_command('update', 'domain_update')
         g.custom_command('unbind', 'domain_unbind')
 
+    with self.command_group('spring-cloud app-insights', is_preview=True, client_factory=cf_spring_cloud_20201101preview) as g:
```

Comment:
only these two commands would use 20201102 api version and all other commands would use another one?

Reply:
only this two commands use 20201101preview api version.